### PR TITLE
added support for _return

### PR DESCRIPTION
**Pull request by Haiyang-Sun**

### CODE REVIEW

File: src/ch.usi.inf.nodeprof.test/js/minitests/fib2.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function fib(n){
+    if (n < 2)
+        return n;
+    return fib(n-1)+fib(n-2);
+}
+
+fib(1);
```
Comment (Haiyang-Sun) [inline src/ch.usi.inf.nodeprof.test/js/minitests/fib2.js]: There is no return event here for fib(1), to be investigated.

Comment (eleinadani) [reply src/ch.usi.inf.nodeprof.test/js/minitests/fib2.js]: any updates on this? Is this a Graal.js problem? If so, could you investigate? If you have time, please go ahead and open a PR on the Graal.js github repo with a fix (or ping me on slack if you have troubles fixing this)

Comment (Haiyang-Sun) [reply src/ch.usi.inf.nodeprof.test/js/minitests/fib2.js]: It's a bug at my part. It should be fixed now. I didn't pay attention to your test code and was using the node return value as the value being returned, which is not the case for FrameReturnNode. Now I change it to use the first input and it works now.

**Comment by alexjordan** [comment]
@Haiyang-Sun can you please squash the 3 commits and rebase on master.

**Comment by alexjordan** [comment]
@Haiyang-Sun I think you rebased on an older master (I synced the repo with internal yesterday) so I fixed that.

**Pull request merged**